.eslintrc.js

```diff
@@ -41,6 +41,15 @@ module.exports = {
   'rules': Object.assign({},
     todo,
     {
+      // turned off otherwise non-admin modules will complain about importing components from admin
+      // via the novel silverstripe js component sharing setup
+      'import/no-extraneous-dependencies': [
```

Comment:
That bit is fine. I was getting that on some of my personal project as well, was wondering where in was coming from.